### ISSUE TITLE
ssh seems to redownload proxy everytime (which is slow)

### ISSUE THREAD

**Issue by taooceros** [lapce/lapce#2476]
## Lapce Version

0.2.8

## System information

Windows 11 -> Arch Linux

## Describe the bug
Remote connection is very slow. After some investigation, it seems like the proxy is redownloaded everytime. 

The script has a line to check the version. However, it seems like the version check has some issue.

by running `/tmp/lapce-proxy.sh v0.2.8 ~/.local/share/lapce-stable/proxy`, the script gives the following message

```
+ lapce_installed_ver='Lapce-Stable 0.2.8'
+ printf '[DEBUG]: Current proxy version: %s\n' 'Lapce-Stable 0.2.8'
[DEBUG]: Current proxy version: Lapce-Stable 0.2.8
+ printf '[DEBUG]: New proxy version: %s\n' v0.2.8
[DEBUG]: New proxy version: v0.2.8
+ '[' 'Lapce-Stable 0.2.8' = v0.2.8 ']'
+ printf 'Proxy outdated. Replacing proxy\n'
```
I am not sure whether this the correct way to execute the script though.

However, the error seems to be at the `cut`. 

`lapce_installed_ver=$("${lapce_dir}/lapce" --version | cut -d'v' -f2)` 

I don't understand why `cut -d'v'` is used here, as the `lapce --version` produce `Lapce-Stable 0.2.8` not `Lapce-Stable v0.2.8`. Also, it seems like the current version contains the `v` when checking.

**Comment by panekj** [comment]
Duplicate of #2202

**Comment by taooceros** [comment]
I will check the prerelease as the pr is merged.

**Comment by taooceros** [comment]
nightly seems to not working at all? The title bar is not wroking, and ssh connection is completely not working. Open a file seems to crash the program.
![image](https://github.com/lapce/lapce/assets/45326534/a876888d-4340-4fd8-92e7-39f7025530e2)

**Comment by alex450** [comment]
Duplicated

**Comment by alex450** [comment]
https://github.com/lapce/lapce/issues/2202#issuecomment-1451585178
this worked

**Comment by panekj** [comment]
Fixed via da1a57c83339633514ffc46012be59f18117ca7d
